pliers/extractors/text.py

```diff
@@ -402,3 +400,47 @@ def _extract(self, stim):
 
         return ExtractorResult(features_list, stim, self,
                                features=self.features, orders=order_list)
+
+
+class WordCounterExtractor(BatchTransformerMixin, TextExtractor):
+
+    ''' Extracts number of times each unique word has occurred within text
+    
+    Args:
+        lemmatize(bool): specifies if words are to be lemmatized before being counted. 
+        log_scale(bool): specifies if count values are to be returned in log scale (defaults to False)
```

Comment:
I would argue this is also not necessary. The user can easily transform this afterwards.

Reply:
added it because suggested in #340